src/temp/ProjectInfo/ExternalCertification.tsx

```diff
@@ -0,0 +1,62 @@
+import { ProjectExpense } from '@planet-sdk/common';
+import styles from './ProjectInfo.module.scss';
+import { useTranslation } from 'next-i18next';
+import DownloadIcon from '../icons/DownloadIcon';
+import SingleProjectInfoItem from './SingleProjectInfoItem';
+import ExternalCertificationItem from './ExternalCertificationItem';
+import ProjectSpendingCertificationItem from './ProjectSpendingCertificationItem';
+import ProgressReportCertificationItem from './ProgressReportCertificationItem';
+
+export const renderDownloadIcon = () => {
+  return (
+    <div className={styles.downloadIcon}>
+      <DownloadIcon
+        width={10}
+        color={`${'rgba(var(--certification-background-color-new))'}`}
+      />
+    </div>
+  );
+};
```

Comment:
Let's split this out into a separate component in a separate file, with a `<button>` at the outermost level instead of a `<div>` - call it `DownloadButton`. Update imports accordingly